src/app/tests/suites/certification/ci-pics-values

```diff
@@ -1521,6 +1521,15 @@ DRLK.S.A0035=0
 #ReadOnly attributes
 DRLK.S.A0028.ReadOnly=1
 DRLK.S.A002c.ReadOnly=1
+DRLK.S.A0080.ReadOnly=0
```

Comment:
Don't these need to be added in the section above under "Attributes" and actually be set to "1" here because they are in fact readonly?

Reply:
The PICS are now added in the attribute section